### PR TITLE
This adds issue and PR templates (#59) fixes #56

### LINKED ISSUE #56: Create Issues And Pull Request Templates
To remove the discrepancy between how issues and bugs are created and to enforce a standard across the same, we should have templates for Issues and PRs. This will help in a standarisation of bug reporting and patching.

Below are two resources which can be used for looking for suitable templates:

- https://github.com/stevemao/github-issue-templates
- https://help.github.com/en/github/building-a-strong-community/creating-a-pull-request-template-for-your-repository

In the bug report, it is advisable to have the following fields:
- [ ] Summary
- [ ] Actual Behaviour
- [ ] Expected Behaviour
- [ ] Steps To Reproduce
- [ ] Screenshots, if applicable
- [ ] Other Relevant Information (System info, browser etc)

In the PR, it is advisable to have the following fields:
- [ ] Summary
- [ ] Details About The Changes
- [ ] Screenshots, if applicable
- [ ] Bugs Introduced By The Patch
- [ ] Reference Concerned Issues

### PR DESCRIPTION
**Pull request by sahil-shubham**
This is regarding the issue #56

### CODE REVIEW

**Comment by sahil-shubham** [comment]
@thealphadollar please review and let me know if I have to make any changes

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+## Steps to reproduce
+1.  
+2.
+3.
+
+## Screenshots (if appropriate)
+
+
+## Context (Environment)
```
Comment (thealphadollar) [inline .github/ISSUE_TEMPLATE.md]: Do you think we should add details here as to what environment can be? For example, operating system, browser name and version, etc.

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Summary
+
+## Expected Behavior
+
+## Current Behaviour
+
+## Steps to reproduce
+1.  
+2.
+3.
```
Comment (thealphadollar) [inline .github/ISSUE_TEMPLATE.md]: Please add a blank line before `1.`

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# Related Issue
+- the issue goes here
```
Comment (thealphadollar) [inline .github/PULL_REQUEST_TEMPLATE.md]: hmm, we can add a template like `fixes #[ISSUE NUMBER]` as they automatically close the linked issue if the PR is merged.

reference: https://docs.github.com/en/enterprise/2.16/user/github/managing-your-work-on-github/closing-issues-using-keywords

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+# Proposed Changes
+- change 1
+- change 2
+
+# Screenshots (if appropriate)
+
+Original                |         Updated
+:------------------------:|:------------------------:
+** original screenshot ** | ** updated screenshot **|
```
Comment (thealphadollar) [inline .github/PULL_REQUEST_TEMPLATE.md]: This is cool though I'm not sure if images can be added in a way that they are presented side by side on Github. Can you please confirm?

**Review by thealphadollar — CHANGES_REQUESTED** [review]
Hey Sahil, thank you for the PR. This looks good and I've suggested a few changes, see if you'd want to incorporate them.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (sahil-shubham) [reply .github/PULL_REQUEST_TEMPLATE.md]: I added it for the same reason,but I thought the PR opener would write the keyword. But, mentioning **fixes** in the template itself is better. Thank you.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 4 above)
Comment (sahil-shubham) [reply .github/PULL_REQUEST_TEMPLATE.md]: side-by-side

This is how it looks, one can look closer by clicking on the image individually.

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (sahil-shubham) [reply .github/ISSUE_TEMPLATE.md]: added the new issue template with points regarding OS, browser, version and device (if the problem is with a specific smartphone).

File: .github/ISSUE_TEMPLATE.md (hunk 2 above)
Comment (sahil-shubham) [reply .github/ISSUE_TEMPLATE.md]: done, thank you.

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (thealphadollar) [reply .github/ISSUE_TEMPLATE.md]: Thanks, Sahil, it looks good to me.

File: .github/ISSUE_TEMPLATE.md (hunk 2 above)
Comment (thealphadollar) [reply .github/ISSUE_TEMPLATE.md]: Thanks.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 3 above)
Comment (thealphadollar) [reply .github/PULL_REQUEST_TEMPLATE.md]: Cool.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 4 above)
Comment (thealphadollar) [reply .github/PULL_REQUEST_TEMPLATE.md]: Oh! This is so cool, I didn't know about this. Thanks, Sahil!

**Review by thealphadollar — APPROVED** [review]
Thanks a lot, Sahil for the contributions!

**Pull request merged**